app/src/providers/selfClientProvider.tsx

```diff
@@ -82,6 +84,9 @@ export const SelfClientProvider = ({ children }: PropsWithChildren) => {
           analytics().trackEvent(event, data);
         },
       },
+      auth: {
+        getPrivateKey: () => unsafe_getPrivateKey(),
```

Comment:
a side note: what about renaming `unsafe_getPrivateKey` to `unsafe_getPrivateKeyDirectlyFromKeychain`?

I'm doing it for anything that reads directly from keychain / writes to it to make it super clear and explicit as the migration continues